themes/OpenFisca-2022/theme.toml

```diff
@@ -0,0 +1,6 @@
+name = "OpenFisca 2022"
+license = "MIT"
+licenselink = "https://github.com/openfisca/openfisca.org/themes/OpenFisca-2022//blob/master/LICENSE"
+description = "The most widely adopted free and open-source engine to write rules as code"
+homepage = "https://openfisca.org"
```

Comment:
This is actually not the description of the theme, and it could be confusing. I suggest we correct (or remove) it 🙂